azure-service-bus-management/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,61 @@
+= Azure Service Bus Management Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft Azure Service Bus Management (Azure Service Bus Management Connector) enables management integration with Azure Service Bus on the cloud. This connector connects to the Azure REST API and supports management operations for queues, topics, and subscriptions. In addition, you can use the built-in management API to discover Azure Service Bus objects automatically and provision them.
+
+For compatibility information, see the Azure Service Bus Management Connector Release Notes.
+// xref:release-notes::connector/<connector>.adoc[Azure Service Bus Management Connector Release Notes].
```

Comment:
Don't forget to add the link to the release notes. We have to merge release notes FIRST so they will exist in latest when we merge the other docs.